app/frontend/src/router/index.js

```diff
@@ -40,6 +40,14 @@ const router = new VueRouter({
       },
       props: true
     },
+    {
+      path: '/apps/requestAccount',
```

Comment:
This should just be `/requestAccount`. This isn't part of apps flow.

Reply:
good idea